release-please-config.json

```diff
@@ -0,0 +1,45 @@
+{
+  "packages": {
+    "solutions/landing-zone": {
+      "package-name": "landing-zone",
+      "changelog-path": "CHANGELOG.md",
+      "tag-separator": "/",
+      "prerelease": true
+    },
+    "solutions/guardrails": {
+      "package-name": "guardrails",
+      "changelog-path": "CHANGELOG.md",
+      "tag-separator": "/",
+      "prerelease": true
+      
+    },
+    "solutions/landing-zone-v2": {
+      "package-name": "landing-zone-v2",
+      "changelog-path": "CHANGELOG.md",
+      "tag-separator": "/",
+      "prerelease": true
+    },
+    "solutions/org-policies": {
+      "package-name": "org-policies",
+      "changelog-path": "CHANGELOG.md",
+      "tag-separator": "/",
+      "prerelease": true
+    },
+    "solutions/heirarchy": {
+      "package-name": "heirarchy",
+      "changelog-path": "CHANGELOG.md",
+      "tag-separator": "/",
+      "prerelease": true
+    },
+    "solutions/gatekeeper-policies": {
+      "package-name": "gatekeeper-policies",
+      "changelog-path": "CHANGELOG.md",
+      "tag-separator": "/",
+      "prerelease": true
+    },
+
+  },
+  "bootstrap-sha": "638cf1a666e2e05394371b2eb9223fc9d539ada0",
+  "release-type": "krm-blueprint",
```

Comment:
Alain and I would prefer to have this option included after line 43: 

"include-v-in-tag": "false",